omnipaxos/src/sequence_paxos/mod.rs

```diff
@@ -214,23 +214,22 @@ where
     pub(crate) fn resend_message_timeout(&mut self) {
         match &self.state {
             (Role::Leader, phase) => {
-                // Resend AcceptStopSign
+                // Resend AcceptStopSign or StopSign's decide
                 if *phase == Phase::Accept {
-                    // TODO: This is slow. Get stopsign from cache instead.
                     if let Some(ss) = self.internal_storage.get_stopsign() {
-                        let decided_idx = self.internal_storage.get_decided_idx();
-                        if !ss.decided(decided_idx) {
-                            for follower in self.leader_state.get_promised_followers() {
-                                // resend if the follower has not accepted the stopsign
-                                if !ss.decided(self.leader_state.get_accepted_idx(follower)) {
-                                    self.send_accept_stopsign(follower, ss.stopsign.clone(), true);
-                                }
+                        for follower in self.leader_state.get_promised_followers() {
+                            let decided_idx = self.internal_storage.get_decided_idx();
+                            if self.leader_state.get_accepted_idx(follower)
+                                != self.internal_storage.get_accepted_idx()
```

Comment:
Just to understand what we're doing here. Say if the leader has accepted_idx = 10 and decided_idx = 8, and follower has accepted_idx = 9. Is the idea that by resending `Decide<9>` that the follower will detect a sequence break and then catch up?